app/components/ExpirationInput/ExpirationInput.css.js

```diff
@@ -0,0 +1,40 @@
+import { StyleSheet } from 'react-native';
+import { Colors } from '../../theme/theme';
+
+export const ExpirationInputCSS = StyleSheet.create({
+  expirationInput: {
+    flexDirection: 'column',
+  },
+  expirationToggleRow: {
+    flexDirection: 'row',
+    paddingRight: 16,
+    justifyContent: 'space-between',
+    height: 48
+  },
```

Comment:
Are these units for padding/height specific for a specific device or do they work for multiple kinds/sizes of phones?